llvm/include/llvm/SYCLLowerIR/SYCLNativeCPUPipeline.h

```diff
@@ -0,0 +1,6 @@
+#include "llvm/Target/TargetMachine.h"
```

Comment:
License header must be added to all new source files!
Please, add here and to llvm/lib/SYCLLowerIR/PipelineSYCLNativeCPU.cpp.

Reply:
Sorry, I've added the licence headers